### ISSUE TITLE
Add support for word-level comments

### ISSUE THREAD

**Issue by rubiot** [rubiot/ibiblia#26]
Today iBiblia only allows for verse-level comments. It' is desireable to do the same, but at word-level, which after could be exported as notes close to the related word.

**Comment by openbibles** [comment]
I agree. Just like in the official Bible modules. For now, Im just typing the word and adding the notes next to it in the notes section.

**Comment by openbibles** [comment]
The notes section seems to be oversized in my opinion. Probably we can make is simpler like this: 
![image](https://user-images.githubusercontent.com/46296587/52609705-2240a480-2eb9-11e9-8a31-a152046d3eca.png)

**Comment by rubiot** [comment]
I'm not sure that would work. I can make the comment box resizeable, but the problem is the status radio group that already is at its mininum size.

**Comment by openbibles** [comment]
If its possible, you can remove the words next to the radio button and just put them each inside a pop up every time user hovers the mouse over them and align them horizontally instead of vertically.

**Comment by openbibles** [comment]
or this: 
![image](https://user-images.githubusercontent.com/46296587/52693718-7a070a80-2fa2-11e9-8cd1-342ee8649180.png)

**Comment by rubiot** [comment]
I've made the comment textbox resizeable in the latest release.

**Comment by openbibles** [comment]
Yes. I see that. Thanks!

Thanks!
Bro. Edmund
www.malayangbiblia.wixsite.com/free<http://www.malayangbiblia.wixsite.com/free>

________________________________

**Comment by openbibles** [comment]
One more thing, I hope its not too much. The notes fonts I must admit is quite small and I have to move my head closer to the monitor screen to double check my notes in there with this little font. Since the field is now retractable, could we somehow make options to adjust the comments fonts? Thanks so much bro!

**Comment by jgkoehn** [comment]
I do think it would be nice to have a per word comment. Sometimes for indicating variants or for other possible translation options. I know it is not hard to directly edit and add <RF>...<Rf> after a word to get this.

But perhaps "ctrl+alt+right click" could open a pop-up menu with such options as <RF>...<Rf> Note, <FI>....<Fi> not original, etc.. 

Then after these are clicked a small editable popup appears so a user can input this content. Or is prefilled if the content already exists.

**Comment by jgkoehn** [comment]
I could see this as also another way to add additional strongs, lemmas, etc. or customizable toggles

**Comment by jgkoehn** [comment]
Perhaps instead of spending time on per word notes. A help guide with things like <RF>....<Rf> for per word comments and also explain how to do headings, subheadings. directly in the editable text. That way a user has more power and once understood it is not so hard.

**Comment by openbibles** [comment]
> 
> 
> Perhaps instead of spending time on per word notes. A help guide with things like < RF > .... < Rf > for per word comments and also explain how to do headings, subheadings. directly in the editable text. That way a user has more power and once understood it is not so hard.

I think the goal of iBiblia is to simplify things for the module creators. Although these commands are simple enough to understand, it takes some time to study and apply them which can take up some of the time that should be spend on the project itself. On the other hand, the addition of these codes on the destination text adds challenges as well in proof-reading or editing.
